test/e2e/pod_create_test.go

```diff
@@ -405,4 +405,18 @@ entrypoint ["/fromimage"]
 		Expect(check1.ExitCode()).To(Equal(0))
 		Expect(check1.OutputToString()).To(Equal("/fromcommand"))
 	})
+
+	It("podman create pod with slirp network option", func() {
+		name := "test"
+		session := podmanTest.Podman([]string{"pod", "create", "--name", name, "--network", "slirp4netns:port_handler=slirp4netns", "-p", "8082:8000"})
```

Comment:
Can you add the network options to `podman pod inspect` and alter the test to verify that `port_handler` was set using that?

Reply:
Should this be under InfraConfig or another (new?) field?

Reply:
Under InfraConfig is fine

Reply:
Done, thanks!